testUtils/TestSession.js

```diff
@@ -189,26 +216,78 @@ class TestSession {
             table.string('model2_prop1');
             table.integer('model2_prop2');
           })
+          .createTable('model3', table => {
+            table.increments('id').primary();
+            table.string('model3Prop1');
+          })
           .createTable('Model1Model2', table => {
             table.increments('id').primary();
             table.string('extra1');
             table.string('extra2');
             table.string('extra3');
-            table.integer('model1Id').unsigned().notNullable().references('id').inTable('Model1').onDelete('CASCADE').index();
-            table.integer('model2Id').unsigned().notNullable().references('id_col').inTable('model2').onDelete('CASCADE').index();
+            table
+              .integer('model1Id')
+              .unsigned()
+              .notNullable()
+              .references('id')
+              .inTable('Model1')
+              .onDelete('CASCADE')
+              .index();
+            table
+              .integer('model2Id')
+              .unsigned()
+              .notNullable()
+              .references('id_col')
+              .inTable('model2')
+              .onDelete('CASCADE')
+              .index();
           })
```

Comment:
Why did you change all these? Do you have different settings for `prettier` or something?

Reply:
Oh, I think my prettier script doesn't touch this file at all. This is fine :+1: